api-utils/JumpCloud_API_Go_Examples/releaseAllUsers/releaseallusers.go

```diff
@@ -24,41 +38,76 @@ import (
 func main() {
 	var apiKey string
 	var urlBase string
+	var orgId string
 
 	// Obtain the input parameters
 	flag.StringVar(&apiKey, "key", "", "-key=<API-key-value>")
-	flag.StringVar(&urlBase, "url", "https://console.jumpcloud.com/api", "-url=<jumpcloud-api-url>")
+	flag.StringVar(&urlBase, "url", jcapi.StdUrlBase, "-url=<jumpcloud-api-url>")
+	flag.StringVar(&orgId, "org", "", "-org=<organizationID (optional for multi-tenant administrators)")
 	flag.Parse()
 
 	if apiKey == "" {
 		fmt.Println("Usage of ./releaseAllUsers:")
 		fmt.Println("  -key=\"\": -key=<API-key-value>")
 		fmt.Println("  -url=\"\": -url=<jumpcloud-api-url> (optional)")
+		fmt.Println("  -org=\"\": -org=<organizationID> (optional for multi-tenant administrators>")
 		return
 	}
 
-	jc := jcapi.NewJCAPI(apiKey, urlBase)
+	if urlBase != jcapi.StdUrlBase {
+		fmt.Printf("URL overridden from: %s to: %s\n", jcapi.StdUrlBase, urlBase)
+	}
+
+	if orgId != "" {
```

Comment:
With these 2 if statements we seem to always print the same string, unless I missed something small.  :-)